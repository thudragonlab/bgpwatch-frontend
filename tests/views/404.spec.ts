import {
  Wrapper,
  shallowMount
} from '@vue/test-utils'
import _404 from "@/views/404.vue";


describe('404.vue', () => {
  it('renders correctly', () => {
  
    const wrapper:Wrapper<any> = shallowMount(_404, {
      mocks: {
      }
    })

    expect(wrapper.html()).toMatchSnapshot();
  });
  
})