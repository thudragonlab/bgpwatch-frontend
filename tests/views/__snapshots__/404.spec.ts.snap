// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404.vue renders correctly 1`] = `<div>404ÔºÅ</div>`;
